nilearn/plotting/matrix_plotting.py

```diff
@@ -37,6 +40,151 @@ def fit_axes(ax):
         ax.set_position(new_position)
 
 
+def _sanitize_inputs_plot_matrix(mat_shape, tri, labels, reorder, figure, axes):  # noqa
+    """Helper function for plot_matrix.
+
+    This function makes sure the inputs to plot_matrix are valid.
+    """
+    _sanitize_tri(tri)
+    labels = _sanitize_labels(mat_shape, labels)
+    reorder = _sanitize_reorder(reorder)
+    fig, axes, own_fig = _sanitize_figure_and_axes(figure, axes)
+    return labels, reorder, fig, axes, own_fig
+
+
+def _sanitize_figure_and_axes(figure, axes):
+    """Helper function for plot_matrix."""
+    if axes is not None and figure is not None:
+        raise ValueError(
+            "Parameters figure and axes cannot be specified "
+            "together. You gave 'figure=%s, axes=%s'" % (figure, axes)
+        )
+    if figure is not None:
+        if isinstance(figure, plt.Figure):
+            fig = figure
+        else:
+            fig = plt.figure(figsize=figure)
+        axes = plt.gca()
+        own_fig = True
+    else:
+        if axes is None:
+            fig, axes = plt.subplots(1, 1, figsize=(7, 5))
```

Comment:
Why is (7, 5) a good figsize ?

Reply:
I don't know why this was chosen as a default. 
To make it a little less hidden in the code, I can add a global constant DEFAULT_FIGSIZE at the beginning of the file and add a note in the docstring saying that if no figure is provided, a new one will be created with this default size.